01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,5 @@
+var main = function() {
+    $('.hidden').fadeIn(5000);
```

Comment:
jQuery fadeIn method provides a workaround (with a second parameter) for callback methods, take a look into jQuery docs http://api.jquery.com/fadein/ :). Please refactor the code to handle the focus inside a callback (learn how to use callbacks, this is really important into JavaScript lenguage)